charts/core/README.md

```diff
@@ -26,7 +26,7 @@ import { Chart, Line, Grid, XAxis, YAxis } from '@lg-charts/core';
 <Chart>
   <Grid vertical={false}>
   <XAxis type="time" />
-  <YAxis type="value" unit="GB" />
+  <YAxis type="value" formatter="{value}GB" />
```

Comment:
this api does feel a little tied to echarts - should we leverage a more expected API and then normalize it on our side before sending thru

Reply:
Certainly open to discussion here, just wanted to call it out

Reply:
Agreed! I'm thinking that we just publicly expose the callback. Addressing this more in [this thread](https://github.com/mongodb/leafygreen-ui/pull/2536#discussion_r1826138443)

Reply:
awesome